modules/ROOT/pages/metrics-list.adoc

```diff
@@ -751,6 +775,707 @@ This metric is a gauge.
+|The time elapsed since the start of the JVM.
+This metric is a gauge.
+|`JVM`
+|{base-metric-features}
+|{mp-1-0}
+
+|memory.committedHeap / (bytes)
+|memory_committedHeap_bytes{mp_scope="base"} / (bytes)
+|The amount of memory that is committed for the JVM to use.
+This metric is a gauge.
+|Base metric
+|{base-metric-features}
+|{mp-1-0}
+
+|memory.maxHeap / (bytes)
+|memory_maxHeap_bytes{mp_scope="base"} / (bytes)
+|The maximum amount of heap memory that can be used for memory management. This metric displays `-1` if the maximum heap memory size is undefined. This amount of memory is not guaranteed to be available for memory management if it is greater than the amount of committed memory.
+This metric is a gauge.
+|Base metric
+|{base-metric-features}
+|{mp-1-0}
+
+|memory.usedHeap / (bytes)
+|memory_usedHeap_bytes{mp_scope="base"} / (bytes)
+|The amount of used heap memory.
+This metric is a gauge.
+|Base metric
+|{base-metric-features}
+|{mp-1-0}
+
+|requestTiming.activeRequestCount
+|requestTiming_activeRequestCount{mp_scope="vendor"}
+|The number of servlet requests that are currently running.
+This metric is a gauge.
+|`RequestTiming`
+|{mp-2-0} or later and feature:requestTiming[display=Request timing]
+|{mp-2-0}
+
+|requestTiming.hungRequestCount
+|requestTiming_hungRequestCount{mp_scope="vendor"}
+|The number of servlet requests that are currently running but are hung.
+This metric is a gauge.
+|`RequestTiming`
+|{mp-2-0} or later and feature:requestTiming[display=Request timing]
+|{mp-2-0}
+
+|requestTiming.requestCount
+|requestTiming_requestCount_total{mp_scope="vendor"}
+|The number of servlet requests since the server started.
+This metric is a counter.
+|`RequestTiming`
+|{mp-2-0} or later and feature:requestTiming[display=Request timing]
+|{mp-2-0}
+
+|requestTiming.slowRequestCount
+|requestTiming_slowRequestCount{mp_scope="vendor"}
+|The number of servlet requests that are currently running but are slow.
+This metric is a gauge.
+|`RequestTiming`
+|{mp-2-0} or later and feature:requestTiming[display=Request timing]
+|{mp-2-0}
+
+|REST.request 
+|REST_request_seconds_max{class="<fully_qualified_class_name>",method="<method_signature>",mp_scope="base"} {empty}+
+ {empty}+
+REST_request_seconds_sum{class="<fully_qualified_class_name>",method="<method_signature>",mp_scope="base"} {empty}+
+ {empty}+
+REST_request_seconds_count{class="<fully_qualified_class_name>",method="<method_signature>",mp_scope="base"} {empty}+
+ {empty}+
+REST_request_seconds{
+ class="<fully_qualified_class_name>",
+ method="<method_signature>",
+ mp_scope="base",
+ quantile=["0.5"\|"0.75"\|"0.95"\|"0.98"\|"0.99"\|"0.999"]
+ }
+|The number of invocations and total response time of this RESTful resource method since the start of the server. The metric will not record the elapsed time nor count of a REST request if it resulted in an unmapped exception. Also tracks the highest recorded time duration and the 50th, 75th, 95th, 98th, 99th and 99.9th percentile.
```

Comment:
```suggestion
|The number of invocations and total response time of this RESTful resource method since the start of the server. The metric does not record the elapsed time nor count of a REST request if it resulted in an unmapped exception. Also tracks the highest recorded time duration and the 50th, 75th, 95th, 98th, 99th and 99.9th percentile.
```